addons/udes_delivery_control/models/picking_extra_fields.py

```diff
@@ -27,44 +52,28 @@ class StockPickingExtras(models.Model):
         readonly=True,
         help="The amount of time taken to complete unloading in HH:MM format",
     )
-    vehicle_type = fields.Many2one("stock.picking.vehicle.type")
-    trailer_number = fields.Integer()
     lane_number = fields.Char()
     pallet_count = fields.Integer(default=0)
     stillage_count = fields.Integer(default=0)
     box_count = fields.Integer(default=0)
-    is_planned = fields.Boolean(
-        "Planned", help="Indicates whether the door control was planned or not", default=False,
-    )
-    is_late = fields.Boolean(
-        "Late", help="Indicates whether the unloading was late if booking slot set", default=False,
-    )
-    is_fast_track = fields.Boolean(
-        "Fast Track", help="Indicates whether the unloading has been fast tracked", default=False,
-    )
 
     # Back Loading Fields
-    is_backload = fields.Boolean(
-        string="Backload Required ?",
-        help="Indicate whether a backload is required. \
-            (e.g. to take back cages used for past deliveries)",
-        default=False,
-    )
+    is_backload = fields.Boolean(compute="_compute_loading_type")
```

Comment:
Do we want readonly?

Reply:
Aren't computed fields readonly by default?

Reply:
Yep computed fields are automatically readonly by default.